elements/nuxeo-suggester/nuxeo-suggester.js

```diff
@@ -136,6 +136,8 @@ Polymer({
 
       #searchBar {
         height: 53px;
+        position: relative;
+        top: var(--nuxeo-app-top, 0);
```

Comment:
we should remove the `top` propriety applied here and move it to its parent (`#suggester`)
 ```suggestion
#suggester {
        top: var(--nuxeo-app-top, 0);
        left: 0;
        position: fixed;
        z-index: 1001;
        width: 100%;
        height: 100%;
        @apply --layout-vertical;
        @apply --layout-center;
      }
```

We should also remove the `position` propriety

Reply:
great suggestion for the suggester :) done.